cmd/sdk-server/main.go

```diff
@@ -36,15 +38,16 @@ import (
 )
 
 const (
-	port = 59357
+	grpcPort = 59357
+	httpPort = 59358
 
 	// specifically env vars
 	gameServerNameEnv = "GAMESERVER_NAME"
 	podNamespaceEnv   = "POD_NAMESPACE"
 
 	// Flags (that can also be env vars)
 	localFlag                  = "local"
-	addressFlag                = "address"
+	addressFlag                = "Address"
```

Comment:
What's the rationale for this change?

Reply:
whoops. Probably a refactor gone awry. Rolled back.